cmd/server/server.go

```diff
@@ -1,9 +1,9 @@
 package main
 
 import (
-	handlers "github.com/nhatvu148/business-day-go/handlers"
+	api "github.com/nhatvu148/business-day-go/api"
```

Comment:
api is not needed here

```suggestion
	"github.com/nhatvu148/business-day-go/api"
```